dali/dfu/dfuwu.cpp

```diff
@@ -416,6 +416,87 @@ static bool notePublisherSubTaskState(const char *subtaskWuid, DFUstate state)
     return true;
 }
 
+static bool prepareToResubmitFailedPublisherTasks(const char *publisherWuid, StringArray &taskWuidsToResubmit)
+{
+    //make sure wuid is a publisher, but not a child task of a publisher
+    if (!publisherWuid || 'P'!=*publisherWuid || strchr(publisherWuid, 'T')!=nullptr)
+       throw MakeStringException(-1,"DFUWU: Attempt to resubmit publisher workunit with non publisher wuid %s", publisherWuid);
```

Comment:
trivial: indented by 3 rather than 4.  Same for lines below.

Reply:
trivial: makeStringException/makeStringExceptionV preferred over MakeStringException

(and others below)